source/manual/redirect-routes.html.md.erb

```diff
@@ -7,17 +7,131 @@ parent: "/manual.html"
 related_applications: [short-url-manager]
 ---
 
-Sometimes, there is a need to manually redirect existing URLs to another
-internal location, usually due to content being archived or because the
-slug has been changed.
+There is a regular requirement for creating short URLs on behalf of
+departments, usually for use with for campaign materials. Content
+designers do this using the [Short URL Manager][short-url-manager] app.
+Read [more about Short URL Manager][short-url-manager-section] below,
+including how to remove routes originating from it.
 
-There is also a regular requirement for creating short URLs on behalf of
-departments, usually for use with for campaign materials.
+There are also processes for [redirecting HRMC manuals][hmrc-manuals-section]
+and [redirecting from campaign sites][campaign-sites-section].
 
-## Getting information on current redirects
+Finally, sometimes there is a need to manually redirect existing URLs to
+another internal location, either due to an incident, content being
+archived or because the slug has been changed.
 
-To find if there is already a redirect for a particular path, open a router-api
-console:
+## Choosing a redirect strategy
+
+Does something exist at the incoming URL already? There's a
+[simple way of checking][check-route-exists-section].
+
+If no route exists, and if you have the time to wait for a content
+designer to review the proposal, you should just use Short URL Manager.
+If the route already exists, or if you're in a hurry, you'll need to take
+a different approach.
+
+The preferred method of creating redirects is to use the publishing app that
+originally published the content to set up the redirect. This usually
+involves unpublishing the content, whereupon the app gives the option to
+redirect the old route to another route. This approach is preferred as it
+keeps the workflow and publishing history within the app, as well as
+correctly propagating the changes through Publishing API, Content Store and
+Router API.
+
+Not all applications support unpublishing / setting redirects. In these
+cases, you should change the content item in Publishing API manually.
+
+### Manually apply a redirect in Publishing API
```

Comment:
I think this section is useful to have as it explains how things work inside the Publishing API, but I suspect the simpler option in most cases is to unpublish from the publishing app itself [but using the API directly](https://github.com/alphagov/publishing-api/blob/master/docs/api.md#post-v2contentcontent_idunpublish). For example, doing something like this:

```rb
GdsApi.publishing_api.unpublish(
  content_id,
  type: "redirect",
  explanation: "manually redirected by YOUR_NAME",
  alternative_path: "/new-path",
  discard_drafts: true
)
```